tests/p2p/integration_test_helpers.py

```diff
@@ -31,9 +31,8 @@ def __init__(self,
                  network_id: int,
                  privkey: datatypes.PrivateKey,
                  ) -> None:
-        min_peers = 1
         discovery = None
-        super().__init__(peer_class, chaindb, network_id, privkey, discovery, min_peers)
+        super().__init__(peer_class, chaindb, network_id, privkey, discovery)
```

Comment:
We should use max_peers=1 here so that it doesn't unnecessarily try to connect to more peers